src/NUnitFramework/framework/Internal/ExceptionHelper.cs

```diff
@@ -34,7 +34,7 @@ namespace NUnit.Framework.Internal
     /// </summary>
     public class ExceptionHelper
     {
-#if !NET45 && !NETSTANDARD1_6
+#if NET20 || NET35 || NET40
```

Comment:
Probably better to be explicit as to what they support rather than what they don't. Good idea 👍

Reply:
Yep. It's nice to call out "which frameworks are making us do this" :D